src/addons/hwtoken/src/main/java/ee/ria/xroad/signer/tokenmanager/module/HardwareModuleWorker.java

```diff
@@ -118,22 +120,29 @@ protected void deinitializeModule() throws Exception {
         }
 
         log.info("Module '{}' got {} slots", module.getType(), slots.length);
+        for (int i = 0; i < slots.length; i++) {
+            log.info("Module '{}' Slot {} ID: {} (0x{})", module.getType(), i, slots[i].getSlotID(),
+                    Functions.toHexString(slots[i].getSlotID()));
+        }
 
-        Map<String, TokenType> tokens = new HashMap<>();
+        // HSM slot ids defined in module data
+        Set<Long> slotIds = module.getSlotIds();
+        log.info("Slot configuration for module '{}' defined as {}", module.getType(), slotIds.toString());
```

Comment:
Could be debug-level logging (otherwise repeated every minute, and most of the time the configuration does not change).